opencti-platform/opencti-front/src/private/components/locations/regions/RegionCreation.tsx

```diff
@@ -94,7 +41,7 @@ const regionMutation = graphql`
   }
 `;
 
-interface RegionAddInput {
+export interface RegionAddInput {
```

Comment:
why the `export` ?

Reply:
That's a leftover from my attempt to generify the CustomFileUploader component.
I have suppressed it in the initial PR.